schemas/v1/6_restore_prove_aggregate_param_schema.go

```diff
@@ -0,0 +1,36 @@
+package v1
+
+func init() {
+	patches.Register(
+		6,
+		`
+	-- Fix ProveCommitAggregate params schema change from Lily v0.8.6
+	-- by updating incorrect JSONB in-place for the affected epochs
+	-- (context: https://github.com/filecoin-project/lily/issues/904)
+
+	SELECT jsonb_set(
+
+		-- given value in column "params"
+		params,
+
+		-- modify the value at key "SectorNumbers"
+		'{SectorNumbers}'::text[],
+
+		-- to use this interpolated json literal
+		concat(
+			'{"rle":', params->'SectorNumbers'->>'RLE',
+			',"elemcount":', params->'SectorNumbers'->>'Count',
+			',"_type":"bitfield"}'
+		)::jsonb,
+
+		-- and do not add if the key is missing
+		false
+
+	)
+	FROM {{ .SchemaName | default "public" }}.parsed_messages
+
+	-- network v15 network upgrade epoch
+	WHERE height > 1594679
+	AND method = 'ProveCommitAggregate';
```

Comment:
This limits the update to only the messages since the upgrade epoch and for this specifically affect message type. The cost will grow the longer that v0.8.6-v0.8.7 is run before updating to v0.8.8.

Reply:
This is slick!
One concern I have: we will need to apply this migration to all messages with params containing bitfields, I think this include more than just the `ProveCommitAggregate` params. Perhaps there is a query that could find all these types?

Reply:
Yes, we can definitely do a manual search to see which types need adjusting and write a migration for each. 👍